src/test/java/io/confluent/connect/hdfs/TestWithMiniDFSCluster.java

```diff
@@ -42,22 +42,41 @@
 import io.confluent.connect.hdfs.filter.TopicPartitionCommittedFileFilter;
 import io.confluent.connect.hdfs.partitioner.Partitioner;
 import io.confluent.connect.storage.common.StorageCommonConfig;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
 
 public class TestWithMiniDFSCluster extends HdfsSinkConnectorTestBase {
 
-  protected MiniDFSCluster cluster;
-  protected FileSystem fs;
+  protected static FileSystem fs;
+  private static MiniDFSCluster cluster;
+
   protected DataFileReader dataFileReader;
   protected Partitioner partitioner;
   protected String extension;
   // The default based on default configuration of 10
   protected String zeroPadFormat = "%010d";
   private Map<String, String> localProps = new HashMap<>();
 
+  @BeforeClass
+  public static void setup() throws IOException {
+    cluster = createDFSCluster();
+    fs = cluster.getFileSystem();
+  }
+
+  @AfterClass
+  public static void cleanup() throws IOException {
+    if (fs != null) {
+      fs.close();
+    }
+    if (cluster != null) {
+      cluster.shutdown(true);
+    }
```

Comment:
```suggestion
    }
    UserGroupInformation.reset();
```

Reply:
To be consistent with https://github.com/confluentinc/kafka-connect-hdfs3/pull/44/files

Reply:
there is no need for this because it is run after the kerberos tests and cleans everything up